kyo-stats-registry/shared/src/main/scala/kyo/stats/internal/UnsafeCounter.scala

```diff
@@ -0,0 +1,19 @@
+package kyo.stats.internal
+
+import java.util.concurrent.atomic.LongAdder
+
+class UnsafeCounter {
+    private var last  = 0L
```

Comment:
Do we need to worry about rollover for long running applications? I realize it's mostly unavoidable.